src/webservice/Router.h

```diff
@@ -14,6 +14,7 @@
 
 #include <proxygen/lib/http/HTTPMethod.h>
 
+#include "common/base/Base.h"
```

Comment:
Needn't depend on this huge header file.

Reply:
FYI. We need not to specify the `common` part in the include path.